api/src/main/java/io/opencensus/trace/propagation/BinaryFormat.java

```diff
@@ -65,23 +65,58 @@
   static final NoopBinaryFormat NOOP_BINARY_FORMAT = new NoopBinaryFormat();
 
   /**
-   * Serializes a {@link SpanContext} using the binary format.
-   *
+   * @deprecated use {@link #toByteArray(SpanContext)}.
    * @param spanContext the {@code SpanContext} to serialize.
    * @return the serialized binary value.
    * @throws NullPointerException if the {@code spanContext} is {@code null}.
    */
-  public abstract byte[] toBinaryValue(SpanContext spanContext);
+  @Deprecated
+  public byte[] toBinaryValue(SpanContext spanContext) {
+    return toByteArray(spanContext);
```

Comment:
There should probably be a note here, too, about subclasses needing to override at least one "to" and "from" method.

Reply:
done.